### PR TITLE
Backend - Update Registrations Document Endpoints

### PR DESCRIPTION
**Pull request by shaangill025**
*Issue:*
- https://github.com/bcgov/entity/issues/22961

*Description of changes:*
- Commented out the registration document endpoints as they will not be used:
    - `GET registrations/<registration_id>/documents`
    - `GET registrations/<registration_id>/documents/<document_id>`
    Needs to be cleanup in future
- Updated `registrations/<registration_id>/documents/<document_id>/file` to `registrations/<registration_id>/documents/<file_key>`
- Added postman test

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by shaangill025** [comment]
image

File: strr-web/composables/useRegistrations.ts [hunk 1]
```diff
@@ -36,9 +36,10 @@ export const useRegistrations = () => {
     axiosInstance.get(`${apiURL}/registrations/${id}`)
       .then(res => res.data)
 
-  const getDocumentsForRegistration = (id: string): Promise<DocumentI[]> =>
-    axiosInstance.get(`${apiURL}/registrations/${id}/documents`)
-      .then(res => res.data)
+  // TODO Remove this
```
Comment (shaangill025) [inline strr-web/composables/useRegistrations.ts:39]: @dimak1 I did this to get the registration-details page working. I couldn't leave it uncommented as the url pattern for other endpoint is also the same. I believe you will be removing this as part of the UI changes.

**Comment by shaangill025** [comment]
3 new SonarCloud issues are all related to TODO comments, items marked for future cleanup.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-116-pxseswpl.web.app

**Pull request merged**